lib/spack/spack/cmd/style.py

```diff
@@ -73,8 +72,10 @@ def __call__(self, fun):
         return fun
 
 
-def changed_files(base=None, untracked=True, all_files=False):
+def changed_files(base=None, untracked=True, all_files=False, root=None):
```

Comment:
This is a bug I also encountered for spack-bot - if develop has not been fetched, you you get this error.

```
$ bin/spack style --root .
fatal: ambiguous argument 'develop...': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
==> Error: Command exited with status 128:
'/usr/bin/git' 'diff' '--name-only' '--diff-filter=ACMR' 'develop...'
```
So probably we should either handle that differently, or do a more specific error message.

Reply:
good point!